controllers/postgres/postgres_controller.go

```diff
@@ -82,7 +83,11 @@ func New(mgr manager.Manager) (*PostgresReconciler, error) {
 	if err != nil {
 		return nil, err
 	}
-	providerList := []providers.PostgresProvider{openshift.NewOpenShiftPostgresProvider(client, clientSet, logger), awsPostgresProvider}
+	providerList := []providers.PostgresProvider{
+		openshift.NewOpenShiftPostgresProvider(client, clientSet, logger),
+		awsPostgresProvider,
+		gcp.NewGCPPostgresProvider(client),
```

Comment:
Could you please look at previous comment  - to validate  Openshift and GCP BlobStorage Providers same way as awsPostgresProvider. And also - error to be informative.